### PR TITLE
Source Chameleon: Fix survey_responses schema and resize icon to 250x250

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Fixes schema and icon
Discussion reference: https://github.com/airbytehq/airbyte/pull/45658#issuecomment-2368985233 

## How
Fixes schema and reference

### CODE REVIEW

File: airbyte-integrations/connectors/source-chameleon/manifest.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: 4.6.2
+version: 5.7.5
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-chameleon/manifest.yaml:1]:
```suggestion
version: 4.6.2
```

**Review by natikgadzhi — APPROVED** [review]

**Review by natikgadzhi — APPROVED** [review]

**Comment by nataliekwong** [comment]
Just tested this, and wanted to share that it resolved the sync issues! Thank you!

**Comment by btkcodedev** [comment]
Anytime! 🙇

**Pull request merged**